matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/internal/crypto/UnwedgingTest.kt

```diff
@@ -173,25 +141,18 @@ class UnwedgingTest : InstrumentedTest {
                 bobSession.cryptoService().getMyDevice().identityKey()!!
         )
         olmDevice.clearOlmSessionCache()
-        Thread.sleep(6_000)
```

Comment:
we should avoid any thread sleeps and instead rely on dynamic updates from `retryPeriodic`, `waitFor`, `suspendCancellableCoroutine`